pegasus/sites.v3/hourofcode.com/public/css/front-page.scss

```diff
@@ -37,10 +37,11 @@ section {
     }
   }
 
-  h3 {
+  h3, h4 {
```

Comment:
Is there any other `h4` content on this page that could be affected?

Reply:
Ah good reminder, thanks for calling this out! Just checked and it's affecting the Highlights box hovers w/ Justin Trudeau, Serena Williams, etc — I'll update the new blurb as a `p` re: https://github.com/code-dot-org/code-dot-org/pull/49377#discussion_r1040151911